docs/en/Community-Articles/2023-01-30-How-To-Contribute-To-a-Open-Source-Software/how-can-I-contribute-to-open-source-projects-on-github.md

```diff
@@ -0,0 +1,105 @@
+
+Previous ABP Community Talk on this topic can be found here  https://www.youtube.com/watch?v=Wz4Z-O-YoPg
+
+## GitHub Issues
+You may want to fix a known bug or work on a planned enhancement. See the [issue list](https://github.com/abpframework/abp/issues) on GitHub.
+
+## Feature Requests
+If you have a feature idea for the framework or modules, create an issue on GitHub or attend an existing discussion. Then you can implement it if it's embraced by the community.
+
+## How to Contribute to a Open Source Software?
+There are some steps to contribute to OSS projects. You can follow the steps below.
+
+## Step 1: Fork the Project
+
+The first thing we need to do now is to fork the open source project. Forking will create a copy of the project in your own GitHub account. This will allow users to make changes to the code without affecting the original repository. Just press the fork key in the project.
+
+![fork-image](images/fork-project-image.png)
+
+After forking, it will create a new repo in your own GitHub profile.
+
+![fork-image-profile](images/fork-project-profile.png)
+
+## Step 2: Clone the Project
+
+In order to develop the project, you need to clone it to your local. After clicking on the code button, select your preferred cloning method and copy the link. You can run the copied link on your local machine with the `git clone` command, but we will use GitHub Desktop. Press `Open with Github Desktop` and the repo will be installed on your local machine. 
+
+or alternatively use the `git clone https://github.com/username/abp.git` command
+
+![clone-image](images/clone-image.png)
+
+## Step 3: Create a New Branch
+
+In this step, you need to create a new branch of your own before you start developing it. Open the repo on GitHub Desktop and create a new branch. When creating a new branch, be careful which branch you create it on.
+
+or alternatively use the `git checkout -b new-branch` command
+
+![branch-image](images/branch-image.png)
+
+## Step 4: Development
+
+Choose a suitable IDE to develop on the new branch you created. In order not to complicate things, we will create a `Developers.md` file and process it. Let's enter a sample text in the Developers file.
+
+![developer-list](images/developer-list.png)
+
+As you can see, all changes made to the repo are reflected directly in Github Desktop.
+
+![github-desktop-change](images/github-desktop-change.png)
+
+## Step 5: Commit
+
+The commit operation is used to save the changes you have made. It is useful to commit after certain operations are done in the project. For the changes you will make, it is useful to write a short sentence describing what you've done. Press the `Commit to <branch-name>` button to commit. 
+
+or alternatively use the `git add .` and `git commit -m "new-branch"` command
```

Comment:
Use `git commit -m "Added the developer list"` instead of `git commit -m "new-branch"`.

Otherwise, it will be different from the picture. See more: https://git-scm.com/docs/git-commit